src/docfx/build/context/BuildScope.cs

```diff
@@ -76,11 +76,17 @@ public bool GetActualFileName(string fileName, out string actualFileName)
             }
         }
 
-        private static Func<string, bool> CreateGlob(Config config)
+        private static Func<string, bool>[] CreateGlob(Config config)
```

Comment:
The return value can still be `Func<string, bool>`

```suggestion
        private static Func<string, bool> CreateGlob(Config config)
```